src/app/api/kamu.graphql.ts

```diff
@@ -0,0 +1,376 @@
+// THIS FILE IS GENERATED, DO NOT EDIT!
+import { gql } from '@apollo/client/core';
```

Comment:
File with interfaces should be named like: `kamu.graphql.interface.ts `

Reply:
Please rename file as you see fit before merging (in `gql-codegen.yml`) as I don't quite understand the naming scheme you're using.

As I mentioned above don't think I can make codegen produce interfaces so `*.interface.ts` might not be appropriate.